DataConnectors/HOLMSecurity/azuredeploy_Connector_HOLMSecurityAssets_AzureFunction.json

```diff
@@ -0,0 +1,216 @@
+              "apiVersion": "2018-11-01",
+              "name": "[variables('FunctionName')]",
+              "location": "[resourceGroup().location]",
+              "dependsOn": [
+                  "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('FunctionName')))]",
+                  "[resourceId('Microsoft.Web/serverfarms', variables('FunctionName'))]",
+                  "[resourceId('Microsoft.Insights/components', variables('FunctionName'))]"
+              ],
+              "kind": "functionapp",
+              "identity": {
+                  "type": "SystemAssigned"
+              },
+              "properties": {
+                  "name": "[variables('FunctionName')]",
+                  "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('FunctionName'))]",
+                  "httpsOnly": true,
+                  "clientAffinityEnabled": true,
+                   "alwaysOn": true
+              },
+              "resources": [
+                  {
+                      "apiVersion": "2018-11-01",
+                      "type": "config",
+                      "name": "appsettings",
+                      "dependsOn": [
+                          "[concat('Microsoft.Web/sites/', variables('FunctionName'))]"
+                      ],
+                      "properties": {
+                          "FUNCTIONS_EXTENSION_VERSION": "~3",
+                          "FUNCTIONS_WORKER_RUNTIME": "python",
+                          "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName')), '2015-05-01').InstrumentationKey]",
+                          "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('FunctionName')), '2015-05-01').ConnectionString]",
+                          "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                          "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                          "WEBSITE_CONTENTSHARE": "[toLower(variables('FunctionName'))]",
+                          "workspaceID": "[parameters('WorkspaceID')]",
+                          "workspaceKey": "[parameters('WorkspaceKey')]",
+                          "logAnalyticsUri": "[variables('LogAnaltyicsUri')]",
+                          "token": "[parameters('Token')]",
+                          "WEBSITE_RUN_FROM_PACKAGE": "<add link to zip>"
```

Comment:
Add WEBSITE_RUN_FROM_PACKAGE link as https://aka.ms/sentinel-holmsecurityassets-functionapp .

Reply:
@NikTripathi Hi!  After adding links, KqlValidations failed. Can you please give me some review?

Reply:
KqlValidations are failing for some other reasons. We are working on it to get it fixed.